src/main/java/uk/gov/ons/census/casesvc/model/dto/Receipt.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ons.census.casesvc.model.dto;
+
+import java.time.LocalDateTime;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class Receipt {
+  private String case_id;
```

Comment:
Not camel case. Can we not use some kind of `@Json` Jackson unmarshalling annotation to get around this horribleness?

Reply:
use `@JsonProperty("case_id")` instead of breaking code style rules